### DISCUSSION TITLE
Not working PBS channels still not identified as [Offline]

### DISCUSSION THREAD

**Discussion by what-a** [iptv-org/community#195] [Q&A] [answered]
Pull request [#5829](https://github.com/iptv-org/iptv/pull/5829) was submitted in an attempt to identify a bunch of not working PBS channels as "[Offline]".  The pull request implemented but the channels are still not labeled as "[Offline]." Any suggestions?

**Comment by BellezaEmporium** [comment, marked as answer]
Do you appear to see the PBS channels on your playlists ?

Because if you do, this is a really abnormal situation, because the bot actually **takes off** the links if said to be offline.

**Comment by what-a** [comment]
Yes, the offline PBS channels are still showing up in the finished playlists.  It appears that their naming reverted from "[Offline]" to "[Geo-blocked]." Perhaps a bug?

**Reply by what-a** [reply]
All fixed now. Thank you to whomever (or the bot) that fixed it.
👍

**Reply by BellezaEmporium** [reply]
> All fixed now. Thank you to whomever (or the bot) that fixed it.
> 👍

One PR made by @freearhey fixed the issue. Props to him.
